processing/src/main/java/org/apache/druid/segment/incremental/IncrementalIndex.java

```diff
@@ -785,9 +780,29 @@ public int size()
     return numEntries.get();
   }
 
-  public long getBytesInMemory()
+  public boolean getDeserializeComplexMetrics()
   {
-    return bytesInMemory.get();
+    return deserializeComplexMetrics;
+  }
+
+  public boolean getReportParseExceptions()
+  {
+    return reportParseExceptions;
+  }
+
+  public AtomicInteger getNumEntries()
+  {
+    return numEntries;
+  }
+
+  public AggregatorFactory[] getMetrics()
```

Comment:
Do all of these getters need to be public ? I think except `getBytesInMemory()`, others can be default package access.

Reply:
You're right.  I saw that they were being used in the index benchmarks but the benchmarks are in the same package.